src/unitxt/operators.py

```diff
@@ -537,7 +541,10 @@ def process(
                     old_value = self.get_default
 
             with error_context(
-                self, field=from_field, action="Process Field", value=old_value
+                self,
+                field=from_field,
+                action="Process Field",
+                value=_limited_repr(old_value),
```

Comment:
```suggestion
```
Not sure about this, I think the error context should point on where there error was not what the cause is, this should be the role of the error itself. my fear is that it will get too busy to read and hide the actual error.